src/main/java/frc/robot/RobotContainer.java

```diff
@@ -1,5 +1,44 @@
 package frc.robot;
 
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.auto.NamedCommands;
+
```

Comment:
Did VSCode add all of theses imports. Right below them are the same imports.

Reply:
Yes VSCode did, I didn't

Reply:
Lmao